### PR TITLE
Add missing command to Automated Update

### PR DESCRIPTION
**Pull request by borzaka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,10 @@ Add this line at the end of the file:
 
 CTRL + X then Y and Enter
 
-`sudo ./whitelist.sh`
+```
+cd whitelist/scripts
```
Comment (SuperSandro2000) [inline README.md]:
```suggestion
sudo whitelist/scripts/whitelist.sh
```

**Comment by anudeepND** [comment]
I think it is better to remove `sudo ./whitelist.sh`  because the cron will execute the script and user need not have to manually execute the script, what do you think @borzaka @SuperSandro2000 ?

**Comment by borzaka** [comment]
That's true, but for the first time it won't be executed right away. The user has to wait until the last day of the week.
I think on the first install user wants to use the list, and not wait.

**Comment by anudeepND** [comment]
@borzaka `sudo whitelist/scripts/whitelist.sh` is enough to run the script, `sudo ./whitelist.sh` is not necessary

**Comment by borzaka** [comment]
@anudeepND Not from the "For Automated Update" section, because there is no `cd whitelist/scripts` instruction there. So you need to change directory, or run the script by include the directory.

**Pull request merged**